src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = []; // Масив для зберігання всіх живих тварин
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this); // Додаємо нову тварину до масиву живих
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      Animal.alive = Animal.alive.filter((animal) => animal !== this);
```

Comment:
The `checkHealth` method correctly removes the animal from the `alive` array if its health is zero or below. This logic is consistent with the task requirements.